packages/shared/src/components/banners/personalized/UserPersonalizedBanner.tsx

```diff
@@ -0,0 +1,36 @@
+import { useQuery } from '@tanstack/react-query';
+import React, { type ReactElement } from 'react';
+import { getBasicUserInfo } from '../../../graphql/users';
+import { AuthenticationBanner, OnboardingHeadline } from '../../auth';
+import { ProfilePicture } from '../../ProfilePicture';
+
+const UserPersonalizedBanner = ({
+  userId,
+}: {
+  userId: string;
+}): ReactElement => {
+  const { data: user, isError } = useQuery({
+    queryKey: ['user_personalized_banner', userId],
```

Comment:
We have a function called `generateQueryKey`, kindly use that here to keep it uniform. If the key is not found yet, feel free to add it on the enum parameter.